app/views/admin_invoices/show.html.erb

```diff
@@ -1,6 +1,18 @@
 <br>
 <br>
-<h6>Invoice id: <%= @invoice.id %></h6>
-<h6>Invoice Status: <%= @invoice.status %></h6>
-<h6>Invoice Created Date: <%= @invoice.date_conversion %></h6>
-<h6>Customer: <%= @invoice.customer_name %></h6>
\ No newline at end of file
+<h3>Invoice id: <%= @invoice.id %></h3>
+<h3>Invoice Status: <%= @invoice.status %></h3>
+<h3>Invoice Created Date: <%= @invoice.date_conversion %></h3>
+<h3>Customer: <%= @invoice.customer_name %></h3>
+<br>
+<h3>Items:</h3>
+<dl>
+<% @items.each do |item| %>
```

Comment:
this is where I would call @invoice.items if you went with the method I described in my earlier comment.